frontend/ui/package.json

```diff
@@ -8,6 +8,9 @@
     "start": "rollup --config -w"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.4",
```

Comment:
Are the version of these dependencies updated? Agus made some fixes here to have all the environment working